Documentation/VMR-re-bootstrapping.md

```diff
@@ -31,10 +31,11 @@ dependency on a new toolset feature until after that feature has been released.
 1. Update previous source-build artifacts
     1. Find a [dotnet-source-build](https://dev.azure.com/dnceng/internal/_build?definitionId=1219)
     with the desired changes.
-    1. Retrieve the built SDKs from the following legs:
+    1. Retrieve the built SDKs and PSB artifacts archives, from the following legs:
```

Comment:
PSB should spelled out.  To me PSB means previous SB artifacts but in this context you are really retrieving the current artifacts to use as the PSB when building the next release.

Reply:
Makes sense - will update to `private source-built artifacts archive`

Reply:
Fixed with https://github.com/dotnet/source-build/pull/4284/commits/035e76da2c943f00e5e65988fc78b987bf675197